modules/container_runtime/variables.tf

```diff
@@ -24,3 +24,83 @@ variable "enable_container_insights" {
 variable "environment_name" {
   type = string
 }
+
+variable "cluster_type" {
+  description = "Select ECS_FARGATE for cluster type as FARGATE, or select ECS_EC2 for cluster type as EC2."
+  type        = string
+  default     = "ECS_FARGATE"
+
+  validation {
+    condition     = contains(["ECS_FARGATE", "ECS_EC2"], var.cluster_type)
+    error_message = "Only 'ECS_FARGATE', and 'ECS_EC2' are allowed."
+  }
+}
+
+variable "az_count" {
```

Comment:
Most of the variables are nowhere used. Please remove all unused variables.

Reply:
Deleted.